setup.md

```diff
@@ -2,13 +2,46 @@
+
+3.  We will be recording one another teaching in pairs or threes during the class, so please bring a cell phone or hand-held camera that can record audio and video. It doesn't have to be high-quality, but it should be good enough that you can understand what someone is saying.
+
+4.  Please also bring a laptop powerful enough to teach on so that you can take part in all of the practical exercises (a tablet won't be sufficient).
+
+Please note that after this course is over, you will be asked to do three short follow-up exercises online in order to finish qualifying as an instructor: the details are available at {{ page.training_site }}/checkout/.  If you have any questions about the workshop, the reading material, or anything else, please get in touch.
+
+If you are interested in doing more reading, you may enjoy:
+
+* *[How Learning Works](http://www.amazon.com/How-Learning-Works-Research-Based-Jossey-Bass/dp/0470484101/)*, which is an excellent summary of current research in teaching and learning
+
+* *[Building a Better Teacher](http://www.amazon.com/Building-Better-Teacher-Teaching-Everyone/dp/0393081591/)*, a well-written look at why educational reforms in the past 50 years have mostly missed the mark and about what we should be doing instead.
+
+* *[Teaching What You Don't Know](http://www.amazon.com/Teaching-What-You-Dont-Know/dp/0674066170/)*, which is a situation many of us find ourselves in more often that we'd like.
+
+Episodes
+--------
+
+Please read through *one* of the episodes below carefully, so that you can do some exercises based on it on the first day of the class.
+
+**Data Carpentry**
+
+* Faceting and Clustering in OpenRefine: <http://www.datacarpentry.org/OpenRefine-ecology-lesson/01-working-with-openrefine.html>
+* Basic Queries in SQL: <http://www.datacarpentry.org/sql-ecology-lesson/01-sql-basic-queries.html>
+* Starting with Data in R: <http://www.datacarpentry.org/R-ecology-lesson/02-starting-with-data.html>
+* Starting with Data in Python: <http://www.datacarpentry.org/python-ecology-lesson/01-starting-with-data>
+
+**Software Carpentry**
+
+* Working with Files and Directories in the Unix Shell: <http://swcarpentry.github.io/shell-novice/02-create.html>
+* Tracking Changes in Git: <http://swcarpentry.github.io/git-novice/04-changes.html>
+* Selecting Data in SQL: <http://swcarpentry.github.io/sql-novice-survey/01-select.html>
+* Repeating Actions with Loops in Python: <http://swcarpentry.github.io/python-novice-inflammation/02-loop.html>
+* Exploring Data Frames in R: <http://swcarpentry.github.io/r-novice-gapminder/05-data-structures-part2.html>
```

Comment:
These look good - but I think we also need to make changes to the lessons where the exercises happen. Should be minor re-wording to change from something like "a question about your research topic" to "a question based on your selected lesson episode". Let me know if you want help making these changes.